colorcet/tests/test_matplotlib.py

```diff
@@ -44,5 +46,9 @@ def test_matplotlib_default_colormap_plot_kbc():
 
 @pytest.mark.parametrize('k,v', list(cc.cm.items()))
 def test_get_cm(k, v):
-    import matplotlib.cm as mcm
-    assert mcm.get_cmap('cet_' + k) is v
+    if sys.version_info < (3, 7):
```

Comment:
In https://matplotlib.org/devdocs/api/next_api_changes/deprecations/23668-TC.html they refer to Matplotlib 3.6 and 3.7, without mentioning any Python version. Why this Python version check?

Reply:
Matplotlib 3.4 only supports Python 3.7, so that's why you get Matplotlib 3.3.4 with Python 3.6. But that doesn't mean you couldn't have installed Matplotlib 3.3 with Python 3.7.